### ISSUE TITLE
Duplicate scenario names causes no tests to run and to exit successfully

### ISSUE THREAD

**Issue by jbridger** [hrcorval/behavex#86]
**Describe the bug**
Having duplicate scenario names when running with parallel test exceution by scenario causes this error in the output:

```
************************************************************
There are duplicate scenario names to run.
Parallel test execution by scenario cannot be performed.
Duplicated scenario names: 
[
    "My duplicate scenario name"
].

************************************************************
```

The exit status is `0`, which means this mistake can quite easily go by undetected.

**To Reproduce**
* Create two or more scenarios with the same scenario name
* Run behavex with parallel schema of `scenario` and with 2 or more processes
* You will see the warning in the log
* Check the exit status code. It should be `0`

**Expected behavior**
The exit status could be non-zero.

Otherwise it leads to a false sense of the tests passing when in reality no tests have been run.

**Desktop (please complete the following information):**
 - OS: MacOS 13.4 and Debian 11.6

**Comment by anibalinn** [comment]
Thanks @jbridger for posting this. We have made the corresponding changes in the upcoming release branch [release_2.0.2](https://github.com/hrcorval/behavex/tree/release_2.0.2)

**Comment by jbridger** [comment]
> Thanks @jbridger for posting this. We have made the corresponding changes in the upcoming release branch [release_2.0.2](https://github.com/hrcorval/behavex/tree/release_2.0.2)

Great to hear, thanks 🙂

**Comment by anibalinn** [comment]
Fix released in version 3.0.0: https://pypi.org/project/behavex/3.0.0/
branch [release_3.0.0](https://github.com/hrcorval/behavex/tree/release_3.0.0)
Thanks!
